README.md

```diff
@@ -2,29 +2,27 @@
-Obviously this is bad for an automated system and it should deal with these issues themselve. Especially when we talk about backup software.
+The local lock system cirumvents the issue of [stale](https://github.com/borgbackup/borg/issues/813) [lock files](https://github.com/borgbackup/borg/issues/2306). When the backup process is interrupted it can happen, that the remote borg repository stays locked. That's why further backups will fail.
+Obviously this is bad for an automated system, especially when talking about backup software.
 
-This issue [has been addressed](https://github.com/borgbackup/borg/pull/1674) in borg **v1.1.0** (currently beta), but I have not tested it and until it works there this here is my workaround.
+The issue [has been addressed](https://github.com/borgbackup/borg/pull/1674) in borg **v1.1.0** (currently beta), but I have not tested it and until it works there, here is my workaround.
 
-Basically the local lock system writes it's PID into a `/var/run` dir (configurable in `RUN_PID_DIR` in [`borgcron.sh`](borgcron.sh#L11)) and as long as the PID is there and the process with this ID is running, the backup is considered to be "locked". When the backup process ends (even when it ends with an error), this lock is removed, so that further backups can start. As the whole thing is done locally, this is considered to be more reliable than the "remote lock" system currently implemented in stable versions of borg.
+Basically the local lock system writes it's PID into a `/var/run` dir (configurable in `RUN_PID_DIR` in [`borgcron.sh`](borgcron.sh#L11)). As long as the PID is there and the process with this ID is running, the backup is considered to be "locked". When the backup process finishes (even if ending with an error), the lock is removed, allowing further backups to start. The whole thing is done locally, thus being more reliable than the "remote lock" system currently implemented in stable versions of borg.
 **For the system to work, the `RUN_PID_DIR` must exist and be writable by the user executing the script.** So please create it before executing the script and adjust the permissions.
 
-**Attention:** As the name implies, this system assumes a "single-user" (one client, one server) mode of the borg repository. As the locking is managed locally, it should be satisfied that **only one client** can access respectively accesses the borg repository. Otherwise **data loss may occur**, as this script automatically breaks the remote lock in the borg repository, if it is not locked locally.
+**Attention:** As the name implies, this system assumes, that you access your borg repo in a "single-user" (one client, one server) environment. As the locking is managed locally, you should ensure, that **only one client** may access the borg repository. Otherwise **data loss may occur**, as this script automatically breaks the remote lock in the borg repository, if it is not locked locally.
```

Comment:
Is not "may" a bit too "weak" here? (I mean this has to be ensured by all means as backups break otherwise.) Maybe also just make it simple and use "usual" indicative form ("client accesses…")?